openc3-cosmos-init/plugins/packages/openc3-js-common/src/services/openc3Api.js

```diff
@@ -24,11 +24,15 @@
 */
 
 import axios from './axios.js'
-import { parse, stringify, isSafeNumber } from 'lossless-json'
+import { parse, stringify, isInteger, isSafeNumber } from 'lossless-json'
 
 // parse huge integer values into a bigint, and use a regular number otherwise
 export function customNumberParser(value) {
-  return isSafeNumber(value) ? parseFloat(value) : BigInt(value)
```

Comment:
This was causing an exception because isSafeNumber was false for floating point numbers that can not be represented like 0.66666666666. So then BigInt was trying to create an integer from a float and was failing.